python-lib/dku_config/additional_parameter.py

```diff
@@ -0,0 +1,19 @@
+class AdditionalParameter:
```

Comment:
All the children class of AdditionalParameter could be in the same file because they are really small or in the same subfolder.

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/33/commits/3126d4cfa2202c8d16cf43e4973a8521113d490d